do.py

```diff
@@ -202,6 +323,13 @@ def dist():
     )
     print(os.listdir("dist"))
 
+    # copy requirements.txt to snappi folder
+    base_dir = os.path.abspath(os.path.dirname(__file__))
```

Comment:
had to do this in dist as we should not create a wheel with requirements.txt in it I suppose